css/css-scroll-snap/support/scroll-target-snap-001-iframe.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<title>iframe for #target and snap position with snapping on</title>
+<style type='text/css'>
+  html, body {
+    margin: 0; padding: 0;
+  }
+  html {
+    /* to make failing more obvious */
+    background: 0 1em / 100% 1em linear-gradient(red, red) repeat-x fixed;
+    /* avoid anti-aliasing issues */
+    font: 20px/1 sans-serif;
+
+    /* turn on snapping */
+    scroll-snap-type: block;
+  }
+  div {
+    height: 1em;
+  }
+  #target { scroll-margin:     1em 0 0;
```

Comment:
Shouldn't that be `scroll-margin:    0 0 1em` ?

Reply:
Ditto